src/node/internal/public_process.ts

```diff
@@ -32,10 +34,103 @@ import {
 
 export { platform, nextTick, emitWarning, env, features };
 
-// TODO(soon): Implement stdio along with TTY streams (and as a requirement for removing experimental).
-export const stdin = undefined;
-export const stdout = undefined;
-export const stderr = undefined;
+/**
+ * Mock non-TTY TTY ReadStream
+ */
+class ReadStream extends Readable {
+  fd: number = 0;
+  isTTY: boolean = false;
+  isRaw: boolean = false;
+  setRawMode(_mode: boolean): this {
+    return this;
+  }
+}
+
+export const stdin = new ReadStream({
+  read(): void {
+    this.push(null);
+  },
+});
```

Comment:
In Node.js, `process.stdin`, `process.stdout`, and `process.stderr` are all *both* `stream.Readable` and `stream.Writable` instances. It is possible (weirdly and not-without-odd-side-effects) to attach `data` events to `process.stdin` and to call `process.stdin.write(...)`, etc.

Reply:
Not to contradict myself ;-) ... but, it is a bit more complicated than that, of course https://nodejs.org/docs/latest/api/process.html#processstdout

> The process.stdout property returns a stream connected to stdout (fd 1). It is a [net.Socket](https://nodejs.org/docs/latest/api/net.html#class-netsocket) (which is a [Duplex](https://nodejs.org/docs/latest/api/stream.html#duplex-and-transform-streams) stream) unless fd 1 refers to a file, in which case it is a [Writable](https://nodejs.org/docs/latest/api/stream.html#writable-streams) stream.

Reply:
I've gone with the `SyncWriteStream` approach instead here now which aligns with a `nohup node app.js` workflow.